packages/react-aria-components/test/Table.test.js

```diff
@@ -981,47 +981,6 @@ describe('Table', () => {
       await user.keyboard('{ArrowUp}');
       expect(document.activeElement).toBe(within(rows[1]).getAllByRole('gridcell')[0]);
     });
-
-    it('should throw error if number of cells do not match column count', () => {
-      expect(() => render(
```

Comment:
you should be able to test it if you structure it like so https://github.com/adobe/react-spectrum/pull/7679/files#diff-b48b45a852f6b7e1b28d59a2757ccaf6409f4472e4d67c9cc68611610a77b3dfR296

Reply:
That was clever, looks like it fixed the problem.